docs/book/en/src/dev/js/modules.md

```diff
@@ -47,12 +47,11 @@ hello_mod_2()
 
 Here is the command to run the demo and the output.
 
-```
-wasmedge --dir .:. target/wasm32-wasi/release/wasmedge_quickjs.wasm example_js/module_demo/demo.js
+```bash
+$ wasmedge --dir .:. target/wasm32-wasi/release/wasmedge_quickjs.wasm example_js/module_demo/demo.js
```

Comment:
Some of your codes add `$ ` and the others remove `$ `.

Reply:
It's the markdownlint rule:
https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md014